### ISSUE TITLE
deno 2.5.2

### ISSUE THREAD

**Issue by Jobians** [mkslanc/ace-linters#195]
deno 2.5.2 LSP error check doesn't work again

**Comment by mkslanc** [comment]
Hi! I need more context to solve that

**Comment by Jobians** [comment]
I use https://github.com/mkslanc/websockets-ls
```
{
  endpointName: 'typescript',
  args: ['deno', ['lsp']],
  nameEndsWith: '.ts',
  connectionType: 'stdio',
  relativePath: false
},
```

With Deno 2.5.2 installed 
Auto completion is working but it doesn't show any errors 

![Image](https://github.com/user-attachments/assets/89c53885-f4c3-49e4-a0b2-03ef81312f27)

**Comment by mkslanc** [comment]
Seems it was because of wrong capability in client. Now it should be fixed -> https://github.com/mkslanc/ace-linters/releases/tag/v1.8.4
I'll do follow-up PR after to support [Pull diagnostics capability](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_pullDiagnostics) in client. But right now it should work with deno. 

<img width="954" height="234" alt="Image" src="https://github.com/user-attachments/assets/95be73cd-52d5-407a-8fd0-f4856c5dae31" />
